x-pack/plugins/synthetics/public/legacy_uptime/components/monitor_management/monitor_list/monitor_detail_flyout.tsx

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiBasicTable,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiErrorBoundary,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiHealth,
+  EuiLoadingSpinner,
+  EuiSelect,
+  EuiSpacer,
+  EuiSwitch,
+  EuiTitle,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import moment from 'moment';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { FETCH_STATUS, useFetcher } from '@kbn/observability-plugin/public';
+import { DecryptedSyntheticsMonitorSavedObject } from '../../../../../common/types';
+import { fetchPings, getMonitor } from '../../../state/api';
+import { ClientPluginsStart } from '../../../../plugin';
+import { useStatusByLocation } from '../../../../hooks/use_status_by_location';
+// TODO: remove any
+const FlyoutHeadingTable = styled<any>(EuiBasicTable)`
+  .euiTableRowCell {
+    border-bottom: 1px solid transparent;
+    border-top: 1px solid transparent;
+  }
+  .euiTable {
+    width: inherit;
+  }
+`;
+
+const BoldItem = styled(EuiFlexItem)`
+  font-weight: bold;
+`;
+
+const STATUS_COLUMN_NAME = i18n.translate('xpack.synthetics.monitorList.statusColumnName', {
+  defaultMessage: 'Status',
+});
+
+const LOCATION_COLUMN_NAME = i18n.translate('xpack.synthetics.monitorList.locationColumnName', {
+  defaultMessage: 'Location',
+});
+
+const DURATION_HEADER_TEXT = i18n.translate('xpack.synthetics.monitorList.durationHeaderText', {
+  defaultMessage: 'Duration',
+});
+
+const MONITOR_DETAILS_HEADER_TEXT = i18n.translate(
+  'xpack.synthetics.monitorList.monitorDetailsHeaderText',
+  {
+    defaultMessage: 'Monitor Details',
+  }
+);
```

Comment:
Please move these to the bottom of file, we have been following this pattern to keep i18n in end of file, to make component more readable.

Reply:
Sure, hadn't really intended for this to be reviewed yet 😆